cligen.h

```diff
@@ -38,6 +38,10 @@
    You include cligen.h which includes the other cligen include files for you.
```

Comment:
These changes are enough. All other changes are simply not needed. I don't understand what is the reason for them.